packages/roosterjs-editor-api/lib/format/clearBlockFormat.ts

```diff
@@ -23,6 +23,8 @@ const ATTRIBUTES_TO_PRESERVE = ['href', 'src'];
 const TAGS_TO_STOP_UNWRAP = ['TD', 'TH', 'TR', 'TABLE', 'TBODY', 'THEAD'];
 
 /**
+ * @deprecated
+ * Use clearFormat passing the FormatType.Autodetect as parameter

```

Comment:
please move the implementation of this function into clearFormat, and call into that function so this function becomes almost empty

Reply:
Alright